internal/kafka/internal/services/kafkatlscertmgmt/kafka_tls_certificate_management_service.go

```diff
@@ -3,13 +3,18 @@ package kafkatlscertmgmt
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/config"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/logger"
 	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/shared"
 	"github.com/caddyserver/certmagic"
 	"github.com/libdns/route53"
+	"go.uber.org/zap"
 )
 
+const certFailedEvent = "cert_failed"
```

Comment:
Can we call it certmagicCertFailedEvent?

Reply:
thanks, changed!